example/README.md

```diff
@@ -1,11 +1,11 @@
-The Example Folder
+# The Example Folder
 This folder must contain a fully working example of the plugin usage. The example must define the required_plugins block.
 The folder can contain multiple HCL2 compatible files. This exapmles should be tested with packer init -upgrade . and packer build ..
 
-If the plugin requires authentication, the configuration should be set as environment variables. Example:
-      env:
-	export OUTSCALE_ACCESSKEYID=<ACCESS_KEY>
-	export OUTSCALE_SECRETKEYID=<SECRET_KEY>
-	export OUTSCALE_REGION=eu-west-2 # Outscale Region
-	export OMI_NAME=<OMI_NAME>
+If the plugin requires authentication, the configuration should be set as environment variables. 
 
+### env:
+	- export OUTSCALE_ACCESSKEYID=<ACCESS_KEY>
+	- export OUTSCALE_SECRETKEYID=<SECRET_KEY>
+	- export OUTSCALE_REGION=eu-west-2 # Outscale Region
+	- export OMI_NAME=<OMI_NAME>
```

Comment:
Not a big fan of that:
1rst we use `OUTSCALE_ACCESSKEYID` instead of `OSC_ACCESS_KEY`
2nd, the `-` isn't bash, so if someone  copy /past thoses lines, he might copy past the `-` with it.

I would prefer:
```
export OSC_ACCESS_KEY=<ACCESS_KEY>
export OSC_SECRET_KEY=<SECRET_KEY>
export OSC_REGION=eu-west-2
export OMI_NAME=<OMI_NAME>
```